src/test/java/com/synopsys/integration/detect/integration/CompleteRiskReportTest.java

```diff
@@ -64,19 +64,19 @@ public void testRiskReportIsPopulated(final boolean includePath) throws Exceptio
 
         final String projectName = "synopsys-detect-junit";
         final String projectVersionName = "risk-report";
-        final ProjectVersionWrapper projectVersionWrapper = assertProjectVersionReady(projectName, projectVersionName);
+        ProjectVersionWrapper projectVersionWrapper = assertProjectVersionReady(projectName, projectVersionName);
         List<File> pdfFiles = getPdfFiles(reportDirectory);
         assertEquals(0, pdfFiles.size());
-        reportService.createReportPdfFile(reportDirectory, projectVersionWrapper.getProjectView(), projectVersionWrapper.getProjectVersionView());
+        File reportPdf = reportService.createReportPdfFile(reportDirectory, projectVersionWrapper.getProjectView(), projectVersionWrapper.getProjectVersionView());
```

Comment:
We don't clean up the file that this test creates.  Capture the PDF file and delete it at the end of the test to cleanup